adapter_designPattern/adapter_designPattern/Program.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace adapter_designPattern
+{
+    /// The 'Client' class
+    public class ThirdPartyBillingSystem
+    {
+        private ITarget employeeSource;
+
+        public ThirdPartyBillingSystem(ITarget employeeSource)
+        {
+            this.employeeSource = employeeSource;
+        }
+
+        public void ShowEmployeeList()
+        {
+            List<string> employee = employeeSource.GetEmployeeList();
+
+            Console.WriteLine("Employee List : ");
+            foreach (var item in employee)
+            {
+                Console.Write(item);
+            }
+
+        }
+    }
+
+    /// The 'ITarget' interface
+    public interface ITarget
+    {
+        List<string> GetEmployeeList();
+    }
+
+    /// Adaptee class
+    public class HRSystem
+    {
+        public string[][] GetEmployees()
+        {
+            string[][] employees = new string[4][];
+
+            employees[0] = new string[] { "100", "Deepak", "Team Leader" };
+            employees[1] = new string[] { "101", "Rohit", "Developer" };
+            employees[2] = new string[] { "102", "Gautam", "Developer" };
+            employees[3] = new string[] { "103", "Dev", "Tester" };
+
+            return employees;
+        }
+    }
+
+    /// Adapter class
+    public class EmployeeAdapter : HRSystem, ITarget
```

Comment:
you don't have to inherit from the `HRSystem` here, you just have to create instance from it.